src/index.js

```diff
@@ -0,0 +1,174 @@
+//--API--//
+
+function getForecast(coordinates) {
+    console.log(coordinates);
+    let apiKey = "f2a7bf87b777299f2d3968c89592e347";
+    let apiUrl = `https://api.openweathermap.org/data/2.5/onecall?lat=${coordinates.lat}&lon=${coordinates.lon}&appid=${apiKey}&units=metric`;
+    console.log(apiUrl);
+
+    axios.get(apiUrl).then(displayForecast);
+}
+
+function getDataTemp(response) {
+
+    document.querySelector("#city").innerHTML = response.data.name;
+    celciusTemperature = response.data.main.temp;
+    document.querySelector("#res").innerHTML = Math.round(celciusTemperature);
+
+    document.querySelector("#hum").innerHTML = response.data.main.humidity;
+    document.querySelector("#wind").innerHTML = Math.round(response.data.wind.speed);
+    console.log(response);
+    document.querySelector("#description").innerHTML = response.data.weather[0].description;
+
+    document.querySelector("#icon").setAttribute("src", `http://openweathermap.org/img/wn/${response.data.weather[0].icon}@2x.png`);
+    document.querySelector("#icon").setAttribute("alt", response.data.weather[0].description);
+
+    getForecast(response.data.coord);
+}
+
+getData("Kyiv");
+
+function getData(city) {
+    let apiKey = "f2a7bf87b777299f2d3968c89592e347";
+    let apiURL = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${apiKey}&units=metric`;
+
+    axios.get(apiURL).then(getDataTemp);
+}
+
+function submit(event) {
+    event.preventDefault();
+    let cName = document.querySelector("#formGroupExampleInput").value;
+    getData(cName);
+}
+let form = document.querySelector("form");
+form.addEventListener("submit", submit);
+
+//--TEMPERATURE--//
+
+let celtemp = document.querySelector("#cel");
+let fahrtemp = document.querySelector("#fahr");
+let restemp = document.querySelector("#res");
+
+
+
+function showcel(event) {
+    restemp.innerHTML = Math.round(celciusTemperature);
+
+}
+
+function showfahr(event) {
+    restemp.innerHTML = Math.round((celciusTemperature * 9 / 5) + 32);
+
+}
+
+celtemp.addEventListener("click", showcel);
+fahrtemp.addEventListener("click", showfahr);
+
+
+
+//--CURRENT LOCATION BUTTON--//
+
+function showTempPos(position) {
+    let lat = position.coords.latitude;
+    let lon = position.coords.longitude;
+    let apiKey = `f2a7bf87b777299f2d3968c89592e347`;
+    let apiURL = `https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&appid=${apiKey}&units=metric`;
+
+    axios.get(apiURL).then(getDataTemp);
+}
+
+function getCurrentPosition(event) {
+    event.preventDefault();
+    navigator.geolocation.getCurrentPosition(showTempPos);
+}
+
+let locButton = document.querySelector("#location");
+locButton.addEventListener("click", getCurrentPosition);
+
+//--DATA AND TIME--//
+
+let now = new Date();
```

Comment:
It would be great to remove all code from global score and leave single function invocation like main or init to improve readability and prevent global scope polllution.